pom.xml

```diff
@@ -948,7 +948,7 @@
             <dependency>
                 <groupId>org.xerial.snappy</groupId>
                 <artifactId>snappy-java</artifactId>
-                <version>1.1.2.6</version>
```

Comment:
this is going to change a snappy version for entire project (not only for kafka). So IMO it should go in separate commit (if possible).

Reply:
its necessary by kafak update..
created a separated commit